lib/core/src/sdk.rs

```diff
@@ -1462,17 +1512,25 @@ impl LiquidSdk {
                         *fees_sat,
                         asset_id,
                     )?;
-                let mut response = self
-                    .pay_liquid(liquid_address_data.clone(), amount_sat, *fees_sat, true)
-                    .await?;
+
+                let mut response = if asset_pay_fees {
+                    self.pay_liquid_payjoin(liquid_address_data.clone(), amount_sat)
```

Comment:
How are we sure here that the asset_fees estimate is what we are going to pay eventually?

Reply:
Can we fail the process if it doesn't meet the estimation?